### ISSUE TITLE
Trailing comma bug

### ISSUE THREAD

**Issue by gilbert** [zeit/styled-jsx#554]
#### Do you want to request a *feature* or report a *bug*?

Bug

#### What is the current behavior?

Leaving a trailing comma on a selector causes the rule to affect all elements. Example:

```css
table td,
table tbody tr, { /* <-- trailing comma */
  border: 1px solid #ccc;
}
```

#### What is the expected behavior?

Should throw a syntax error since it's not valid css.

#### Environment (include versions)

* OS: OS X Mojave
* Browser: Safari
* styled-jsx (version): 3.2.1

#### Did this work in previous versions?

Not sure

**Comment by giuseppeg** [comment]
CSS is permissive and never throws errors. You might want to use a linter maybe https://github.com/giuseppeg/styled-jsx-plugin-stylelint

**Comment by gilbert** [comment]
Ok fine, but that's not the bug. The bug is "Leaving a trailing comma on a selector **causes the rule to affect all elements**". This is not how CSS behaves.

**Comment by giuseppeg** [comment]
oh fair enough sorry for closing right away. In that case you may want to file an issue to this repo https://github.com/thysultan/stylis.js/issues - this is the CSS preprocessor we use.
